aten/src/ATen/core/ivalue.cpp

```diff
@@ -367,10 +367,6 @@ StrongTypePtr::StrongTypePtr(
   cu_ = std::move(cu);
   type_ = type;
   TORCH_INTERNAL_ASSERT(type_);
-  if (type_->cast<ClassType>()) {
```

Comment:
We can possiblly keep this assert if the class table no longer holds StrongTypePtr objects

Reply:
`ivalue::object::create` requires a `StrongTypePtr` for instantiation of any object, and if it's a custom class that's going to be nullptr, so I don't think we can remove this